README.md

```diff
@@ -136,8 +134,6 @@ sudo openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout /etc/ssl/privat
 
 *Important*: Make sure that the certificate files are named exactly `ssl-certificate.key` and `ssl-certificate.crt`. If you have procured the certificate from a different source, rename them to the prescribed filenames, and ensure that they are located in the same folder as the `install.sh` script. They will get uploaded to the Chef Habitat supervisor during the install.
 
-*Important*: If you get authentication failures with a self-signed cert, you may need to either update the cert package that the Chef Habitat on-prem services are using (in the `/etc/systemd/system/hab-sup.service` file), or modify the hab `core/cacerts` package (in the `/hab/pkgs/core/cacerts/...` folder) to add your self-signed cert chain to the `cert.pem` file. Restart the hab services with `systemctl restart hab-sup` after updating the cert file. The latest version of the Chef Habitat Builder on-prem services will also look for certificates in the `/hab/cache/ssl` directory - you may copy any self-signed certificates to that directory if needed.
```

Comment:
We don't want to remove this paragraph entirely, just update it to copy the cert into the '/hab/cache/ssl' directory instead of modifying the cacerts package.

Reply:
Let's put this into the troubleshooting section. Do we need to keep the `update the cert package` step?

Reply:
yeah, I think we can remove the update cert package step also